include/document.h

```diff
@@ -4,9 +4,19 @@
 #include <memory>
 #include <string>
 
+#if defined(_MSC_VER) && defined(LUNASVG_SHARED)

```

Comment:
What about other compilers?

Reply:
Mingw doesn't seem to need `__declspec(dllexport)`, unless I did something wrong. Let me make sure.

Reply:
Confirmed. Tested with mingw-w64-gcc 10.2.0. Seems like every symbol is exported, just like on Linux.